Userland/Libraries/LibWeb/UIEvents/PointerEvent.cpp

```diff
@@ -27,6 +27,13 @@ PointerEvent::PointerEvent(JS::Realm& realm, FlyString const& type, PointerEvent
     , m_is_primary(event_init.is_primary)
     , m_persistent_device_id(event_init.persistent_device_id)
 {
+    m_coalesced_events.ensure_capacity(event_init.coalesced_events.size());
+    for (auto const& coalesced_event : event_init.coalesced_events)
+        m_coalesced_events.unchecked_append(static_cast<PointerEvent&>(*coalesced_event));
+
+    m_predicted_events.ensure_capacity(event_init.predicted_events.size());
+    for (auto const& predicted_event : event_init.predicted_events)
+        m_predicted_events.unchecked_append(static_cast<PointerEvent&>(*predicted_event));
```

Comment:
Ah, sorry, was just about to merge this, but noticed these casts on a last look - they don't seem necessary? Compiles without the casts for me

Reply:
Thanks, somewhere in swapping out the types I forgot to remove those casts.